### ISSUE TITLE
[Feature Request] Quick Acess to Camera App from Lockscreen

### ISSUE THREAD

**Issue by ivoxavier** [ubports/ubuntu-touch#473]
One of the most handy features that iOS and Android have is the ability to access camera app from the lockscreen without unlock the device.

On our platform we've to unlock the phone first. And in some cases we need the camera to open ASAP.

Thanks.

**Comment by dark-eye** [comment]
Hey  just to contribute as little,  I think this might be a general thing, that the user might want some apps to be opened without password/lock and provide limited access to the storage (something like a session store)

**Comment by ebeton-ro** [comment]
+1

**Comment by samzanemesis** [comment]
You can use the sidebar to highlight the camera icon, would be nice if you didnt need to unlock the phone to open that one app

**Comment by JassMan23** [comment]
@samzanemesis I agree. Since the sidebar itself can be opened without login it should show unavailable apps as greyed out. System settings or UTtweakTool should allow you to select which items are not greyed out and can thus be run without a login.
If the "torch" app could be made to work reliably (maybe just my Nexus5 with a problem) this could be launched from the sidebar thus removing the need to bring up the battery settings to find the torch icon.
